atlasdb-persistent-lock-api/src/main/java/com/palantir/atlasdb/sweeperservice/SweeperService.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.sweeperservice;
+
+import javax.annotation.Nullable;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Provides endpoints for sweeping a specific table.
```

Comment:
Note that this operation can be long running. So if using HTTP/2 and sweeping a table takes up more time than the server's idle timeout, the server will close the stream (but the manually triggered sweep will continue to run) and the client won't get the success notification. 

I'm not sure if this is something we want to address before we recommend people use this without consulting us, though it's something we should be aware of. Not the scope of this PR, but when this merges we should file a ticket noting that this is an issue.